plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -207,6 +214,17 @@ private function get_feature_definitions() {
 					'disable_ui'         => false,
 					'is_legacy'          => true,
 				),
+				'order_source_attribution' => array(
+					'name'               => __( 'Order Source Attribution', 'woocommerce' ),
+					'description'        => __(
+						'Enable this feature to track and credit channels and campaigns that contribute to orders on your site. (Beta)',
+						'woocommerce'
+					),
+					'enabled_by_default' => true,
+					'disable_ui'         => false,
+					'is_legacy'          => true,
```

Comment:
❔ By the definition of this flag, it seems like this should probably be `false`: `Checks whether a feature id corresponds to a legacy feature (a feature that existed prior to the implementation of the features engine).`
```suggestion
					'is_legacy'          => false,
```

But is it `is_experimental`?

Reply:
> By the definition of this flag, it seems like this should probably be false: Checks whether a feature id corresponds to a legacy feature (a feature that existed prior to the implementation of the features engine).

The main issue is the whole plugin compatibility system. By labeling it as legacy, we skip the compatibility checks, which aren't really relevant to this feature.

Reply:
In six months it might be hard to understand this rationale. Ideally we don't introduce unintended workarounds like this, but in the interest of shipping at a reasonable pace let's at least document this _why_ in a comment here.

Reply:
Clarified in e9b6408075ef78b551d3fad266cf4cd52e13c2fe